tests/backup/backup_helper.go

```diff
@@ -1934,7 +1934,11 @@ func UpgradePxBackup(versionToUpgrade string) error {
 	storageClassName := pvcs.Items[0].Spec.StorageClassName
 
 	// Get the tarball required for helm upgrade
-	cmd = fmt.Sprintf("curl -O  https://raw.githubusercontent.com/portworx/helm/%s/stable/px-central-%s.tgz", latestPxBackupHelmBranch, versionToUpgrade)
+	helmBranch, isPresent := os.LookupEnv("PX_BACKUP_HELM_REPO_BRANCH")
+	if !isPresent {
+		helmBranch = latestPxBackupHelmBranch
```

Comment:
lets change this from latestPxBackupHelmBranch to <default/master>PxBackupHelmBranch

Reply:
Changed it to `defaultPxBackupHelmBranch`